job/job.go

```diff
@@ -42,7 +42,7 @@ type Config struct {
 type Object interface {
 	GetMetaObject() metav1.Object
 	GetRuntimeObject() runtime.Object
-	GetK8upStatus() *k8upv1alpha1.K8upStatus
+	GetK8upStatus() *k8upv1alpha1.Status
```

Comment:
I'm not sure about this, but shouldn't it be `GetStatus()` now?

```suggestion
	GetStatus() *k8upv1alpha1.Status
```